code/js/constant/string-attribute-element-constant.js

```diff
@@ -0,0 +1,7 @@
+const STRING_ATTRIBUTE_ELEMENT_CONST = {
```

Comment:
HTML_ELEMENT_ATTRIBUTE_CONST would be better.

Reply:
file renamed according your advice